docs/BR.md

```diff
@@ -1970,13 +2096,14 @@ j. Other Subject Attributes
    Other attributes MAY be present within the subject field. If present, other attributes MUST contain information that has been verified by the CA.
 
 #### 7.1.4.3 Subject Information - Root Certificates and Subordinate CA Certificates
+
 By issuing a Subordinate CA Certificate, the CA represents that it followed the procedure set forth in its Certificate Policy and/or Certification Practice Statement to verify that, as of the Certificate's issuance date, all of the Subject Information was accurate.
 
 ##### 7.1.4.3.1 Subject Distinguished Name Fields
 
-a.  __Certificate Field:__ `subject:commonName` (OID 2.5.4.3)  
-    __Required/Optional:__ Required  
-    __Contents:__ This field MUST be present and the contents SHOULD be an identifier for the certificate such that the certificate's Name is unique across all certificates issued by the issuing certificate.
+a. __Certificate Field:__ `subject:commonName` (OID 2.5.4.3)
+   __Required/Optional:__ Required
```

Comment:
```suggestion
a. __Certificate Field:__ `subject:commonName` (OID 2.5.4.3)  
   __Required/Optional:__ Required  
```

It looks like the trailing space got removed here.

Reply:
Yep, although again, the next ones down (ON and CN) don't have those trailing spaces but appear formatted fine in the current version of the BRs.

Reply:
No, they do. Highlighting in the GitHub web interface shows them too :)